openedx/adg/lms/our_team/__init__.py

```diff
@@ -1,5 +1,6 @@
 """
-init file for our team
+This app adds the ability to manage team members and board members, dynamically. New team and board members
+can be added from Django admin, similarly they can be updated from Django admin. ADG's our team page displays
```

Comment:
Instead of "ADG's our team page", we can write "Omnipreneurship Academy's 'Our Team' page" or "ADG LMS site's 'Our Team' page".